nottrobin/cloud-init.io
# Eleventy fixes

## Done

- Fixed global nav js being ignored in build
- Fixed site metadata not having values
- Replaced head and footer with includes 

## QA

- Run the site with `dotrun`
- Check that the page title shows in the browser tab
- Check that the global nav loads
- Check that the html `head` content is in the source
- Check that the footer is present on the page
